admin/partials/acc_user_importer-admin-settings.php

```diff
@@ -14,7 +14,7 @@
 	 */
 	add_action( 'admin_menu', 'accUM_add_menu_page' );
 	function accUM_add_menu_page () {
-		add_options_page(
+		add_users_page(
```

Comment:
I reverted to how it was, otherwise the "delete" button to delete a log file would not work.